src/cdk/menu/menu-trigger-base.ts

```diff
@@ -36,6 +37,15 @@ export const MENU_TRIGGER = new InjectionToken<CdkMenuTriggerBase>('cdk-menu-tri
   },
 })
 export abstract class CdkMenuTriggerBase implements OnDestroy {
+  /** The DI injector for this component. */
+  readonly injector = inject(Injector);
+
+  /** The view container ref for this component */
+  protected readonly viewContainerRef = inject(ViewContainerRef);
+
+  /** The menu stack in which this menu resides. */
+  protected readonly menuStack: MenuStack | null = inject(MENU_STACK, InjectFlags.Optional);
```

Comment:
Why is this optional now? It used to be required. It causes some extra null checks in the components extending this one.

Reply:
Bad copy and paste, removed the optional